### PR TITLE
Create StaticDiscoveryModule

### PR DESCRIPTION
**Pull request by johngmyers**

### CODE REVIEW

File: configuration/pom.xml [hunk 1]
```diff
@@ -43,6 +43,12 @@
             <version>3.1</version>
         </dependency>
 
+        <dependency>
```
Comment (alinav) [inline configuration/pom.xml]: do we have to bind google auto-value?
Immutables.org might be better? http://marxsoftware.blogspot.com/2016/06/lombok-autovalue-immutables.html

Comment (johngmyers) [inline configuration/pom.xml]: Immutables would have made excluding the `Key` attribute from `equals()` slightly easier, but AutoValue is used extensively elsewhere in Platform.

File: CHANGES [hunk 2]
```diff
@@ -9,6 +9,12 @@ Platform 1.57
   "service-balancer.SERVICE.consecutive-failures" configuration option. A
   value of "5" is recommended.
 
+* Discovery
+
+  We created a StaticDiscoveryModule which does not talk to the discovery
+  service. It instead obtains service location from configuration, using
+  the "service-balancer.SERVICETYPE.uri" config option.
```
Comment (gwittel) [inline CHANGES]: Please include a note that it supports comma separated list of URIs

**Comment by johngmyers** [comment]
Tweaked CHANGES and merged

**Pull request closed without merging**